### DISCUSSION TITLE
mspider/intro/features

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2776] [Announcements]
# mspider/intro/features

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/intro/features.html

**Comment by samzong** [comment]
功能总览页面需要更新，应补充：VM 服务支持、多云互联支持、边车服务发现范围等特色功能点

#2777

**Comment by hanxiaop** [comment]
细粒度访问授权 的描述可以更准确一点

**Comment by kebe7jun** [comment]
对 Dubbo 的说明，应该描述清楚是 Dubbo 协议、Dubbo 框架还是 Dubbo 序列化，，，Dubbo SDK 一般指 Dubbo 框架。
